### PR TITLE
Wrapper for slider 'favourites movies'

### PR DESCRIPTION
**Pull request by solomiia-sahala**
-created wrapper for favourites movies to render movies in slider;
-connected axios;
-added global styles to root;

### CODE REVIEW

File: src/components/WrapperFavorites.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from 'react';
+
+import Title from './Title';
+import Container from './Container';
+import Slider from './Slider';
+import SmallInfoCard from './SmallInfoCard';
+
+import api from 'src/service/api';
```
Comment (Dmytro1991ua) [inline src/components/WrapperFavorites.tsx:8]: Please use relative import, as done in other places (other imports) => import api from '../service/api'

**Review by Dmytro1991ua — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -55,4 +56,4 @@
       "last 1 safari version"
     ]
   }
-}
+}
```
Comment (Kachyr) [inline package.json:59]: add newline

**Review by oleksandrstarov — CHANGES_REQUESTED** [review]
It seems to be some extra changes in this PR, can you check please?

File: src/App.tsx [hunk 3]
```diff
@@ -1,26 +1,27 @@
...
     <>
-      <Layout>
-        <Router>
+      <Router>
+        <Layout>
           <Switch>
-            <Route exact path="/" component={Home} />
-            <Route path="/movie-details/:id" component={MovieDetails} />
-            <Route path="/favorites" component={Favorites} />
-            <Route path="/search-results" component={SearchResults} />
```
Comment (Kachyr) [inline src/App.tsx:20]: Strange, wasn't it already on the main branch?

File: src/axios/url.tsx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import axios from 'axios';
+
+import { API_URL } from '../constants/api';
+
+const url = axios.create({
+    baseURL: `${API_URL}`
+});
+
+export default url;
```
Comment (Kachyr) [inline src/axios/url.tsx:9]: add newline

File: src/components/Favorites.tsx [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
-    <div className="favorites-wrapper">
-      <MoviesList pageTitle="Favorites" />
+    <div className='favorites-wrapper'>
+      <MoviesList pageTitle='Favorites' />
     </div>
   );
 };
 
-export default Favorites;
+export default Favorites;
```
Comment (Kachyr) [inline src/components/Favorites.tsx:11]: add newline

File: src/components/SmallInfoCard.tsx [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+        <div className="small-cardInfo__rate-voters">Voters<br />{data.vote_average}</div>
+      </div>
+      <div className="small-cardInfo__filter"></div>
+      <img src={`https://image.tmdb.org/t/p/w185${data.poster_path}`} alt={data.title} className="smallCardInfo__img" />
+      <link rel="stylesheet" href={`/movie-details/${data.id}`}>
+        <button className="small-cardInfo__button">
+          <p className="small-cardInfo__button-text">VIEW DETAILS</p>
+          <img src={buttonImgSrc} alt="button Img" className="small-cardInfo__button-img" />
+        </button>
+      </link> */}
```
Comment (Kachyr) [inline src/components/SmallInfoCard.tsx:36]: Thats a lot of commented code

File: src/constants/api.tsx [hunk 7]
```diff
@@ -0,0 +1,4 @@
+export const API_KEY = 'api_key=157c2ade7d0b335008ae899a157d8967';
+export const API_URL = 'https://api.themoviedb.org/3/';
+export const API_IMG_URL = 'https://image.tmdb.org/t/p/';
+export const API_DATE_QUERY = 'primary_release_date.lte=';
```
Comment (Kachyr) [inline src/constants/api.tsx:4]: add newline

File: src/service/api.tsx [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+    let obj = await axios.get(`search/movie?${API_KEY}&query=${query}`);
+    return obj.data.results;
+  },
+
+  changeImgLinks(url: string, size: string) {
+    return `${API_IMG_URL}${size}${url}`;
+  },
+}
+
+export default api;
```
Comment (Kachyr) [inline src/service/api.tsx:126]: newline

File: src/styles/Breadcrumbs.scss [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+    &:last-child {
+      color: $breadcrumbGrey;
+      pointer-events: none;
+
+      &:after {
+        content: '';
+      }
+    }
+  }
+}
```
Comment (Kachyr) [inline src/styles/Breadcrumbs.scss:38]: add newline

File: src/styles/Detail.scss [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+
+.detail-title{
+  font-weight: bold;
+}
+
+@media screen and (max-width:992px){
+  .detail-body{
+    font-size:18px;
+  }
+}
```
Comment (Kachyr) [inline src/styles/Detail.scss:15]: add newline

File: src/styles/Header.scss [hunk 11]
```diff
@@ -1,112 +1,159 @@
...
+      display: none;
+
+      @media screen and(min-width: 992px) {
+        display: block;
+        margin-right: 14px;
+      }
+    }
   }
-}
+}
```
Comment (Kachyr) [inline src/styles/Header.scss:159]: add newline

File: src/styles/SmallCardInfo.scss [hunk 12]
```diff
@@ -0,0 +1,94 @@
...
+      color: $white;
+    }
+
+    .small-cardInfo__button-img{
+      height: 15px;
+      width: 15px;
+      margin: 6px 0;
+    }
+  }
+}
```
Comment (Kachyr) [inline src/styles/SmallCardInfo.scss:94]: add newline

**Review by Kachyr — COMMENTED** [review]
check coments

File: src/components/WrapperFavorites.tsx [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+      setMovieInfo(res);
+    });
+  }, []);
+
+  return (!movieInfo.length ? null : (
+    <>
+      <Container>
+        <Title text={'Favorite movies'} />
+        {movieInfo.length && (<Slider>
+          {movieInfo.map((movie: MovieId) => {
```
Comment (oleksandrstarov) [inline src/components/WrapperFavorites.tsx:28]: (movie: MovieId) >> ({ id }: MovieId) so less code :)

File: src/components/WrapperFavorites.tsx [hunk 14]
```diff
@@ -0,0 +1,42 @@
...
+const WrapperFavorites = () => {
+  const [movieInfo, setMovieInfo] = useState<MovieId[]>([]);
+
+  useEffect(() => {
+    api.getDataByIds(api.getFavoritsIdList()).then((res: any) => {
+      setMovieInfo(res);
+    });
+  }, []);
+
+  return (!movieInfo.length ? null : (
```
Comment (oleksandrstarov) [inline src/components/WrapperFavorites.tsx:23]: it is nicer to have check 
if(!movieInfo.length) {
return null;
}

return (markup)

File: src/App.scss [hunk 15]
```diff
@@ -13,6 +13,10 @@ body {
     background-color: $bg-color-light;
 }
 
+#root {
+    overflow: hidden;
+}
+
```
Comment (oleksandrstarov) [inline src/App.scss:19]: Cant really find where this is used....

**Review by oleksandrstarov — APPROVED** [review]
Looks good, nice work!
Check comments

**Pull request merged**